roles/dosomething.varnish/files/503.html

```diff
@@ -0,0 +1,101 @@
+<!DOCTYPE html>
```

Comment:
Let's move this out of role, to the top [`files/`](https://github.com/DoSomething/tower/tree/master/files) folder.